bot/cogs/hacktoberstats.py

```diff
@@ -0,0 +1,188 @@
+import re
+import typing
+from collections import Counter
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+
+class Stats:
+    def __init__(self, bot):
+        self.bot = bot
+        self.channel_id = (496432022961520650)  # Hardcode #event-hacktoberfest channel ID
+
+    @commands.command(
+        name="stats",
+        aliases=["getstats", "userstats"],
+        brief="Get a user's Hacktoberfest contribution stats",
+    )
+    async def get_stats(self, ctx, username: str):
+        """
+        Query GitHub's API for PRs created by a GitHub user during the month of October that
+        do not have an 'invalid' tag
+
+        For example:
+            !getstats heavysaturn
+
+        If a valid username is provided, an embed is generated and posted to the channel
+
+        Otherwise, post a helpful error message
+
+        The first input argument is treated as the username, any additional inputs are discarded
+        """
+        prs = await self.get_october_prs(username)
+
+        post_channel = self.bot.get_channel(self.channel_id)
```

Comment:
Actually, now that I'm reading this, why are we not just posting it to the channel it is requested? We should be replying to `ctx.channel`.

Reply:
Changed